lib/logout.js

```diff
@@ -209,9 +206,9 @@ module.exports.logout = function (options) {
 
       // Logout flow in progress, incoming SAMLResponse from SP. (Could be SP initiated or IdP initiated)
       } else if (req.query.SAMLResponse || req.body.SAMLResponse) {
-        function processLogoutResponse(logoutResponse, transactionId) {
+        function processLogoutResponse(state, transactionId) {
```

Comment:
Shouldn't we check here that the logout response info is the same as we have in the stored state? at least the issuer, it's weird we ignore the response and only use the info from our state